frontends/web/src/components/sidebar/sidebar.css

```diff
@@ -1,3 +1,15 @@
+.closeButton {
+    background: transparent;
+    border: none;
+    display: none;
+    padding: 12px;
+    transform: translateX(14px);
```

Comment:
this is to "negate" the padding above  (otherwise the X will appear a little bit to the left). 

no big deal, but 14 is used here and not 12 (actual padding val) because of the extra whitespace brought in by the actual `CloseXWhite` component itself.